### PR TITLE
Added ability to use with use_frameworks! CocoaPods directive

### PR DESCRIPTION
**Pull request by dmytrozavgorodniy**
- Use use_frameworks option in Podifle
- Rewrite e2e tests
- Use the last version of tipsi-appium-helper module
- Reset iOS simulator each time when run e2e tests

### CODE REVIEW

**Comment by itsmepetrov** [comment]
I think we should export `TPSTwitterModule` as default: 

```
import TPSTwitterModule from './TPSTwitterModule'

export default TPSTwitterModule
```

File: example/__tests__/setup/loginToTwitter.js [hunk 1]
```diff
@@ -0,0 +1,213 @@
...
+        .click(confirmEmailSubmitButtonId)
+    } catch (e) {
+        // Do nothing
+    }
+
+    // Authorize app if needed
+    try {
+      await driver
+          .waitForVisible(authorizeAppButtonId, 30011)
+          .click(authorizeAppButtonId)
```
Comment (itsmepetrov) [inline example/__tests__/setup/loginToTwitter.js]: Remove extra spaces

File: example/__tests__/setup/loginToTwitter.js [hunk 2]
```diff
@@ -0,0 +1,213 @@
...
```
Comment (itsmepetrov) [inline example/__tests__/setup/loginToTwitter.js]: the same

File: example/__tests__/setup/loginToTwitter.js [hunk 3]
```diff
@@ -0,0 +1,213 @@
...
+      await driver
+        .waitForVisible(accessToTwitterAccountsTitleId, 30001)
+        .waitForVisible(accessToTwitterAccountsDeclineButtonId, 30002)
+        .waitForVisible(accessToTwitterAccountsAcceptButtonId, 30003)
+        .click(accessToTwitterAccountsAcceptButtonId)
+    } catch (error) {
+      // Do nothing...
+    }
+
+    //
```
Comment (itsmepetrov) [inline example/__tests__/setup/loginToTwitter.js]: remove this line

File: example/__tests__/setup/loginToTwitter.js [hunk 4]
```diff
@@ -0,0 +1,213 @@
...
+
+  if (platform('ios')) {
+    //
+    // Alert
+    //
+    // Title: “example” Would Like Access to Twitter Accounts
+    //
+    // Button: Don’t Allow
+    // Button: OK
+    //
```
Comment (itsmepetrov) [inline example/__tests__/setup/loginToTwitter.js]: remove this line

File: example/__tests__/setup/loginToTwitter.js [hunk 5]
```diff
@@ -0,0 +1,213 @@
...
+  // Android only
+  const signOutButtonId = idFromXPath(`
+    //android.widget.LinearLayout[1]/android.widget.FrameLayout[1]/
+    android.widget.RelativeLayout[1]/android.webkit.WebView[1]/android.webkit.WebView[1]/
+    android.view.View[1]/android.view.View[2]/android.widget.ListView[1]/android.view.View[1]/
+    android.widget.Button[1]
+  `)
+
+  if (platform('ios')) {
+    //
```
Comment (itsmepetrov) [inline example/__tests__/setup/loginToTwitter.js]: remove this line

File: example/__tests__/setup/loginToTwitter.js (hunk 2 above)
Comment (itsmepetrov) [inline example/__tests__/setup/loginToTwitter.js]: remove extra spaces

File: example/__tests__/setup/loginToTwitter.js [hunk 6]
```diff
@@ -0,0 +1,213 @@
...
+          .click(authorizeAppButtonId)
+    } catch (e) {
+        // Do nothing
+    }
+  }
+
+  if (platform('android')) {
+    //
+    // Sign out Button
+    //
```
Comment (isnifer) [inline example/__tests__/setup/loginToTwitter.js]: please, use single-line comments on single line, not three

File: example/__tests__/setup/loginToTwitter.js (hunk 1 above)
Comment (dmytrozavgorodniy) [inline example/__tests__/setup/loginToTwitter.js]: I removed extra spaces, please check

File: example/__tests__/setup/loginToTwitter.js (hunk 2 above)
Comment (dmytrozavgorodniy) [inline example/__tests__/setup/loginToTwitter.js]: I removed extra spaces, please check

File: example/__tests__/setup/loginToTwitter.js (hunk 5 above)
Comment (dmytrozavgorodniy) [inline example/__tests__/setup/loginToTwitter.js]: I removed extra line, please check

File: example/__tests__/setup/loginToTwitter.js (hunk 4 above)
Comment (dmytrozavgorodniy) [inline example/__tests__/setup/loginToTwitter.js]: I removed extra line, please check

File: example/__tests__/setup/loginToTwitter.js (hunk 3 above)
Comment (dmytrozavgorodniy) [inline example/__tests__/setup/loginToTwitter.js]: I removed extra spaces, please check

File: example/__tests__/setup/loginToTwitter.js (hunk 2 above)
Comment (dmytrozavgorodniy) [inline example/__tests__/setup/loginToTwitter.js]: I removed extra spaces, please check

File: example/__tests__/setup/loginToTwitter.js (hunk 6 above)
Comment (dmytrozavgorodniy) [inline example/__tests__/setup/loginToTwitter.js]: I used single-line comments, please check

**Comment by densa** [comment]
LGTM

**Comment by cybergrind** [comment]
good job :+1:

**Pull request merged**